web/styles/right_sidebar.css

```diff
@@ -82,11 +82,6 @@
         & i {
```

Comment:
It'd be really nice while we're touching this area to put a class on this `i` tag.

Reply:
Used ".zulip-icon-more-vertical" class instead as that conveys for what specific icon we are needing this font tweak.

Reply:
That's not the best answer -- one should prefer a semantic class over one that's what icon we selected. But it's fine for now.